plugins/inputs/ipset/ipset.go

```diff
@@ -133,8 +143,9 @@ func setList(timeout config.Duration, useSudo bool) (*bytes.Buffer, error) {
 func init() {
 	inputs.Add("ipset", func() telegraf.Input {
 		return &Ipset{
-			lister:  setList,
-			Timeout: defaultTimeout,
+			lister:        setList,
+			entriesParser: ipsetEntries{},
```

Comment:
There is no need in doing this initialization as `entriesParser` should already be initialized like this.